### DISCUSSION TITLE
Docs on how to use the "Search logs via query" feature in the Database logs panel

### DISCUSSION THREAD

**Discussion by chipilov** [supabase/supabase#4617] [Questions] [answered]
The database logs panel has a toggle which says "Search logs via query". I assume this allows searching the logs using SQL queries, but anything I write there results in "Sorry! Could not fetch data".

Are there any docs about how to use it?

**Comment by pranavbadami** [comment]
+1 on this, can't seem to query the logs either, even using the "POST or PATCH" templates

**Reply by Ziinc** [reply]
Hi @pranavbadami , the templates have been updated as the "POST and PATCH" template is log source specific. The displayed templates have been fixed to be source aware. 

You may find additional documentation [here](https://supabase.com/docs/guides/platform/logs)

**Comment by Ziinc** [comment, marked as answer]
Hi @chipilov , additional platform documentation on the log browser features have been added, you may reference them [here](https://supabase.com/docs/guides/platform/logs).

Do let me know if you run into any issues relating to it.

**Reply by chipilov** [reply]
@Ziinc I see a couple of issues on my dashboard:
  - The timestamp input field on the top right says "From", however, if I give it a custom timestamp, it seems that it only shows me logs UNTIL the time I enter, not FROM the time I enter;
  - The "10 minutes ago" template does NOT return anything for me even though I know I've had logs in the last 10 minutes;

Let me know if you need more details or if you want me to open separate issues about these.

**Reply by chipilov** [reply]
Also, it seems that either DB logs get rotated very quickly (e.g. less than 1 hr) or there is another issue with timestamp field on the top right because after a little less than 1hr of looking at some logs, they were no longer displayed when I set a time about an hour in the past to view them again.

**Reply by Ziinc** [reply]
Thanks @chipilov for raising the timestamp labelling ux issue. 

DB Logs are currently limited to a 2hr window temporarily, and we are looking to increase this soon.

RE the querying issues that you are facing, you can open up a support ticket through the dashboard with steps to replicate so that I can properly investigate your project issues. Do tag me in it so that it gets routed to me. 

![image](https://user-images.githubusercontent.com/22714384/156012095-ad418c22-62f2-4de0-aa1a-cbb6f3662824.png)

**Comment by mike-luabase** [comment]
@Ziinc  How do you filter for errors?
